src/bookshelf/rebar.config

```diff
@@ -34,8 +34,9 @@
         {git, "git://github.com/chef/mixer.git",                      {tag, "0.1.1"}}},    
     {mochiweb, ".*",
         {git, "git://github.com/mochi/mochiweb.git",                  {branch, "master"}}},
+    % pinning this for now because an update to master (recon dep) broke compilation
     {observer_cli, ".*",
-        {git, "git://github.com/zhongwencool/observer_cli.git",       {branch, "master"}}},
+        {git, "git://github.com/zhongwencool/observer_cli.git",       {ref, "bcc0dfaa2013a436ce7e597731acab6577321b43"}}}, 
```

Comment:
we should create an issue with some more context about the reason for pinning and how we can remove the pin.

Reply:
Will test to see whether this pin can be removed now, but it doesn't appear they've made any changes that would affect this upstream.

Reply:
Will test to see whether this pin can be removed now, but it doesn't appear they've made any changes that would affect this upstream.

Reply:
Seems to no longer be an issue.  Here's an integration test with bookshelf changes stacked on the deps changes, and the observer_cli change (above) reverted:

https://buildkite.com/chef/chef-chef-server-master-integration-test/builds/344